api/models/member.py

```diff
@@ -22,7 +23,16 @@ class Member(Base):
     creation_date = db.Column(db.DateTime())
 
     def get_display_name(self):
-        return "{} {}".format(self.first_name,self.last_name)
+        return "{} {}".format(self.first_name, self.last_name)
+
+    def is_guest(self):
+        return self.role_id == 1
```

Comment:
Where is the mapping for role names to role ids specified ?

Reply:
@sujen1412 good catch--I just added role constants to the Role model.